content/docs/getting-started/futures.md

```diff
@@ -194,7 +194,12 @@ printing a message, here), and retrieve the value later on.
 So far this example isn't terribly interesting, since there are simpler ways to
 work with thread pools. But one strength of futures is their ability to
 *combine*. We'll show this off by combining the thread pool future with a
-timeout future:
+timeout future.
+
+We'll use a timer from the `tokio-timer` crate, which manages its timeouts in a
+separate thread. There are also timeouts in `tokio-core`. They don't need
+another thread, but they need an event loop, something we'll have a look at
+later.
```

Comment:
This seems like it may not be that relevant to explaining of the next example? This is one of the first documents seen by those interacting with features, so we just want to make sure it's as straightforward and as clear as possible.